content-src/components/DebugPage/DebugPage.js

```diff
@@ -45,6 +46,33 @@ const DebugPage = React.createClass({
 
     return (<main className="debug-page">
       <div className="new-tab-wrapper">
+
+        <section className="experiments">
+          <h2>Experiments</h2>
+          <p>
+            To override these experiments, check <code>about:config</code> for <code>extensions.@activity-streams.experiments.prefName</code>, where <code>prefName</code> is the name of the pref in this table.
```

Comment:
Aww. We can't directly link to about:config or the filtered view:
`Security Error: Content at resource://activity-streams/data/content/activity-streams.html#/ may not load or link to about:config?filter=extensions.@activity-streams.experiments.`